client/src/components/Dashboard/Dashboard.css

```diff
@@ -23,24 +22,24 @@
   display: none;
 }
 
-.dashboard .favorites-tab {
+.dashboard .favorites-tab, .followers-tab {
   width: 100%;
   padding: 10px 10px;
   display: none;
 }
 
 .dashboard .carousel {
   width: 100%;
-  background: url('../../../img/lily.jpeg');
+  background: url('/Users/kimlinh/CodePath/Project/Capstone-Project/client/img/lily.jpeg');
   background-size: cover;
   background-position: 5rem;
   height: 35vh;
 }
 
 /* navlinks section */
 .dashboard .nav {
-  margin: 10px 0px;
-  padding: 10px 0px 0px 0px;
+  margin: 10px 0;
+  padding: 10px 0 0 0;
```

Comment:
Add px to all values. There are a few other places in this diff as well that should have px added.
https://www.w3schools.com/css/css_padding.asp